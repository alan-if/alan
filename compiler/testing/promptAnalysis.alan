Prompt
  Say Current Actor.
  Say This.

The loc Isa location
End The loc.

Start At loc.
