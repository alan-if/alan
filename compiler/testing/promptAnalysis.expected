########## promptAnalysis ##########


promptAnalysis.alan

    3.    Say This.
=====>        1

  *1*   421 E : THIS instance is not defined in this context.


        1 error(s).
        No detected warnings.
        1 informational message(s).

